services/api-server/src/simcore_service_api_server/api/routes/solvers.py

```diff
@@ -164,32 +209,33 @@ async def get_solver_release(
             detail=f"Solver {solver_key}:{version} not found",
         ) from err
 
+    else:
+        return solver
```

Comment:
that is the TRY300. I don't find it very nice. I disabled it. so feel free to do it as well.
https://github.com/ITISFoundation/osparc-simcore/pull/4429/files#diff-b3236d4cb260ca6acce083493fcdc42816f732fc3d5aefb263a3c3fe9db4f81f